src/theme/class-theme.php

```diff
@@ -2,20 +2,41 @@
 /**
  * The Theme specific functionality.
  *
+ * @since   4.0.0 Manifest as DI.
  * @since   1.0.0
  * @package Inf_Theme\Theme
  */
 
 namespace Inf_Theme\Theme;
 
 use Eightshift_Libs\Core\Service;
-use Inf_Theme\General\Manifest;
+use Eightshift_Libs\Assets\Manifest_Data;
 
 /**
  * Class Theme
  */
 class Theme implements Service {
 
+  /**
+   * Manifest Data Interface.
+   *
+   * @var Manifest_Data
+   *
+   * @since 4.0.0 Init.
+   */
+  protected $manifest;
+
+  /**
+   * Contruct method
+   *
+   * @param Manifest_Data $manifest Load Manifest data.
```

Comment:
```suggestion
   * @param Manifest_Data $manifest Inject manifest which holds data about assets from manifest.json.
```

Maybe something along these lines?

Reply:
👍